PSKoans/Koans/Introduction/AboutCmdletVerbs.Koans.ps1

```diff
@@ -25,37 +24,38 @@ Describe "Basic Verbs" {
         <#
             Cmdlets with the Get verb are used for retrieving data.
 
-            So for example "Get-Process" will retrieve all the processes running on
-            your current machine.
+            So for example "Get-Process" will retrieve all the processes running
+            on your current machine.
         #>
 
         It 'is for commands that retrieve data' {
             <#
-                Using the Get-Command cmdlet, which retrieves all available commands,
-                find 5 commands with the Get verb.
+                Using the Get-Command cmdlet, which retrieves all available
+                commands, find 5 commands with the Get verb.
 
-                Replace each ____ with the name of a Get-* command.
+                Fill in each blank with the name of a different Get-* command.
             #>
-            $Answers = "____", "____", "____", "____", "____"
+            $Answers = @(
+                "____"
+                "____"
+                "____"
+                "____"
+                "____"
+            )
             $Answers | Should -BeIn (Get-Command -Verb Get).Name
 
-            $Answers | Get-Unique | Should -HaveCount 5 -Because "five unique cmdlets are required"
+            $Answers |
```

Comment:
Why split on the pipe here? 
```suggestion
            $Answers | Get-Unique | Should -HaveCount 5 -Because "five unique. cmdlets are required"
```
is fine and reads easier

Reply:
It's such a long line thooooo